runtime-modules/forum/src/lib.rs

```diff
@@ -929,25 +926,16 @@ decl_module! {
             // Signer does not match creator of post with identifier postId
             ensure!(post.author_id == forum_user_id, ERROR_ACCOUNT_DOES_NOT_MATCH_POST_AUTHOR);
 
-            // Update post text and record update history
+            // Update post text
             <PostById<T>>::mutate(post_id, |p| {
                 let text_hash = T::calculate_hash(&new_text);
```

Comment:
Seeing that this was already done before but in general, computation should not be done inside the `mutation` call if possible, only updating the value should be done in the closure.